apps/web/components/pages/task/details-section/blocks/task-secondary-info.tsx

```diff
@@ -104,23 +106,6 @@ const TaskSecondaryInfo = () => {
 				</ActiveTaskVersionDropdown>
 			</TaskRow>
 
-			{/* Epic */}
```

Comment:
@CREDO23 why this whole block is removed!?

Reply:
It was duplicated @evereq 

![Screenshot from 2024-11-27 19-28-35](https://github.com/user-attachments/assets/9ab16680-e7aa-4b9c-b0e4-bfff40e5990a)